wdl-lsp/src/server.rs

```diff
@@ -398,7 +398,7 @@ impl LanguageServer for Server {
                 references_provider: Some(OneOf::Left(true)),
                 completion_provider: Some(CompletionOptions {
                     resolve_provider: Some(false),
-                    trigger_characters: Some(vec![".".to_string(), " ".to_string()]),
+                    trigger_characters: Some(vec![".".to_string()]),
```

Comment:
Added that for a better ux - like user typing `version` then hitting space would show `1.0`,`1.1`,`1.2`, this is great but it was causing a lot of unwanted completions requests for other sections.

With the current changes. `version` completions would work as:
1.  When user types `version` and triggers auto completion (CTRL+SPACE)
2. When user types `version` and then starts typing a known supported version `1.*`